force_wfmanager/central_pane/result_table.py

```diff
@@ -37,6 +37,10 @@ class ResultTable(HasStrictTraits):
         ))
     )
 
+    def __init__(self, analysis_model):
```

Comment:
One issue with this is that you're breaking the standard traits initialization. I would strongly suggest using keyword arguments for your __init__ if you want to keep it. Doing it will allow you to remove the __init__ method without breaking things.

Make sense? 

Then you could even remove line 42 and just pass the kwargs to the `__init__` call.